server/src/main/java/org/apache/cassandra/sidecar/modules/CassandraOperationsModule.java

```diff
@@ -118,6 +192,14 @@ VertxRoute cassandraSchemaRoute(RouteBuilder.Factory factory,
     }
 
     @Deprecated
+    @GET
+    @Path(ApiEndpointsV1.DEPRECATED_ALL_KEYSPACES_SCHEMA_ROUTE)
+    @Operation(summary = "Get all keyspaces schema (deprecated)",
```

Comment:
Also mark it as deprecated in the spec?
```suggestion
    @Operation(summary = "Get all keyspaces schema (deprecated)",
    deprecated = true, 
```